app/tests/integration/nimbus/jexl/__init__.py

```diff
@@ -44,9 +42,7 @@ def to_str(node):
     elif type(node) == BinaryExpression:
         return f"({to_str(node.left)} {node.operator.symbol} {to_str(node.right)})"
     elif type(node) == Transform:
-        args = ""
-        if node.args:
-            args = f'({", ".join([to_str(a) for a in node.args])})'
+        args = f'({", ".join([to_str(a) for a in node.args])})' if node.args else ""
```

Comment:
what the heck

Reply:
This is a [conditional expression](https://docs.python.org/3/reference/expressions.html#conditional-expressions) which are usually called 'ternarny' expression in some other languages.

Reply:
well I think the ```", "``` part is weird to read. The line just looks a bit confusing to read

Reply:
It is a confusing line, I would generally avoid putting so much logic on a single line even though you can I try to keep it as like 'each line does one thing'.  Also the ", ".join(...) was already there, all its doing here is folding the if statement into a conditional statement?

Reply:
But we can throw this change out, I agree it's a very busy line.